source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -490,6 +490,8 @@ def __init__(self):
                                 dest="add_poi_infos", action="append",
                                 help="Show more information about the poi if it's available, for instance, show "
                                      "BSS/car park availability in the pois(BSS/car park) of response")
+        parser_get.add_argument("no_shared_section", type=BooleanType(), default=False, hidden=False,
```

Comment:
Just a question : don't we want a more generic parameter saying that we are in a mode that manages journeys in a smarter way (grouping, maybe more in the future).
Something that may also deactivate some other params...
And then this one would be hidden and activated through the other param.
If we have a question dangling, it's always easier to change a hidden param to a public one than the other way :)

Reply:
I suggested to discuss this parameter in 'refinement' and stand-up mettings already...
So, what would you suggest?